Dockerfile.frontend

```diff
@@ -1,7 +1,7 @@
 FROM node:18-slim
 LABEL maintainer="hello@wagtail.org"
 
-RUN apt-get update && apt-get install rsync -y
+RUN apt-get update && apt-get install rsync make -y
```

Comment:
Installs `make` on the frontend container so that we are able to run the Wagtail repo's own `make` commands.